.github/buildbox/Vagrantfile

```diff
@@ -29,7 +29,11 @@ EOF
 chmod +x /etc/profile.d/set_build_env.sh
 ENV_VARS_SCRIPT
 
-DISTROS = [ "debian8", "debian9", "debian10", "amazon2", "centos6", "centos7", "centos8", "fedora31", "fedora32" ]
+DISTROS = [ "debian8", "debian9", "debian10",
```

Comment:
Not in scope for this PR probably, but I'm surprised to see this out-dated list here.  I know we can't yet support Fedora 33 or 34, but why keep Fedora 32 and debian8?

Reply:
I'm trying to have wide kernel version matrix by different distros. Fedora 31 is used for the tests on kernel 5.8 and Fedora 32 is used for the tests on kernel 5.9. There are some significant changes between these 2 kernel versions. Please take a look to the PR [Support Linux kernel 5.9 and 5.10](https://github.com/elastio/elastio-snap/pull/75/files). Without fedora 32 I couldn't test these changes. Fedora 31 tests these changes: [Linux kernel 5.8 support](https://github.com/elastio/elastio-snap/pull/72/files) and [Fix tests on the loop dev on Linux 5.8](https://github.com/elastio/elastio-snap/pull/74/files).
And Debian 8 has kernel version 3.16. It's easy to break something for the old kernel versions when adding support of the new kernel versions. So, I'm keeping it for now, to have tests for some other 3.1X kernel except CentOS 7 with the kernel 3.10. Because CentOS's kernels have many unusual backports. And, for instance, XFS inconsistency after restore on the 3.X kernel much better to debug on Debian 8 and then adjust a fix for CentOS 7. Then move on kernels 4.X like Amazon Linux 2 Debian 9. And so on...
Moreover these tests are running very fast comparatively with other our builds and we already have these boxes to run tests on them. We'll not save much compute time by dropping builds for Debian 8 or some Fedora but will loose informative feedback from the tests. For instance, in this PR I know, that my changes for the 5.11 kernel support doesn't break driver on the kernels 5.8, 5.9, 4.16, 3.16 and others.
I hope I answered your question and you stop asking me why these old distros are still here.

~And I'll tell you a secret, locally I also debugged the XFS problem on CentOS 6 with kernel 2.6.32.~ And I could understand the root cause of the problem. And now I have 80% of the fix for the 3.X (Debian 8, CentOS 7) and 4.X (CentOS 8, Amazon Linux 2, Debian 9, 10) kernels. Before that I was trying to resolve the problem on the kernels 5.X and lost much time without meaningful progress. It is much better to move from simple to complex than to break teeth at once on complex.